greenpithumb/camera_manager.py

```diff
@@ -6,6 +6,7 @@
 # Format of filename to write for camera image file (assumes timestamp is in
 # UTC), as YYYY-MM-DDTHH:MMZ (minutes-level precision).
 _FILENAME_FORMAT = '%Y-%m-%dT%H%MZ.jpg'
+LIGHT_THRESHOLD_PCT = 75
```

Comment:
Also, can we add a comment explaining what this is? Also, I think 60% is enough for decent pictures.